extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -157,6 +163,9 @@ export class ConsentUI {
     /** @private {boolean} */
     this.enableBorder_ = DEFAULT_ENABLE_BORDER;
 
+    /** @private {boolean} */
+    this.lightboxEnabled_ = false;
```

Comment:
1. You're mixing naming styles (`enableBorder_` and `lightboxEnabled_`), make them consistent.

2. Explain what `lightbox` means here. The term `lightbox` is already overloaded in our codebase (`amp-lightbox` vs. `amp-lightbox-gallery`), which might make it confusing. Consider using `modal` instead.

Reply:
Changed lightbox => modal